contracts/interfaces/IDrawCalculator.sol

```diff
@@ -51,7 +52,7 @@ interface IDrawCalculator {
      * @notice Read global DrawBuffer variable.
      * @return IDrawBuffer
      */
-    function getPrizeDistributionBuffer() external view returns (IPrizeDistributionBuffer);
+    function getPrizeDistributionSource() external view returns (IPrizeDistributionSplitter);
```

Comment:
Why does the Draw Calculator know about the splitter?

Reply:
Cause we need the `getPrizeDistributions` function from the interface.